### DISCUSSION TITLE
Selection tool

### DISCUSSION THREAD

**Discussion by playgroundearth** [keiwando/toonsquid#575] [Q&A] [answered]
Hello 
The video https://m.youtube.com/watch?v=tHWWa7mVvVM
Shows copy and paste options under the corner menu

on my app I only see cut or copy
I have or see no way to paste copied selections

I have the latest version 1.10.9

I tried with pixel and vector layers

What am I doing wrong?

thank you

**Comment by keiwando** [comment]
Is the animation layer you're trying to paste into maybe locked? If not, could you maybe share a short screen recording that shows the issue and the selected layers?

**Comment by playgroundearth** [comment]
Hello Keiwando

amazing app first of all. I am learning

I was not able to add the video here as they bigger than the limit. I will send over email

thanks

**Comment by playgroundearth** [comment]
Hello Keiwando/Toonsquid

I am attaching a video here
  RPReplay_Final1714338428.MP4
<https://drive.google.com/file/d/1wvcDcEJGeiyb0H0SXyuKs4tE2BpcI1KJ/view?usp=drivesdk>

I was following a video showing how to use the selection tool to select a
part of an image or shape cut, copy and paste.

I am not able to figure out what is wrong with the way I am doing it.

Thank you

Fernanda

**Comment by playgroundearth** [comment]
Hello

I figured it out. The item needs to be selected first. Once selected, with the transform tool. Once the shape or drawing has the box around it, then I can switch to the selection tool and select a part to copy and paste or cut. The transform tool kind of works as a pre select step.

Thanks

**Comment by playgroundearth** [comment, marked as answer]
I figured it out. What I was doing wrong that was not selecting the drawing first. if I select the drawing first using a transform tool, then I can switch to the select tool and it works. The drawing or shape needs to be selected first the transform box needs to appear. then it’s possible to switch to the other tool, the actual select tool, and select a part or piece to copy or paste. Thank you.

**Comment by playgroundearth** [comment]
Hello Keiwando/Toonsquid

I found the problem. I need to first select the image or shape with the
transform tool, which "selects" the image adding the square around it.
Then, I can click the real selection tool and everything works. I can copy,
paste, cut, etc.

I will post this on the comments for others to read.

Thank you and sorry for the distraction

Fernanda
